docs/swagger.yml

```diff
@@ -1505,29 +1506,58 @@ definitions:
             example: 65
   InsightsReport:
     properties:
-      report_id:
-        type: "integer"
-        format: "int64"
-        description: "The insights report identifier"
-        example: 1
-      report_type:
-        type: "string"
-        description: "The report type"
-        example: "insights"
-      report_version:
-        type: "string"
-        description: "The report version"
-        example: "1.0.0.afdsgev"
-      report_platform_id:
-        type: "string"
-        description: "Globally unique uuid for report"
-        example: "be3075ac-84d3-4b62-9f5c-a418a36f802d"
-      hosts:
+      metadata_tar_path:
         type: object
         properties:
-          system_platform_id:
-            $ref: "#/definitions/Host"
-        description: "The insights report for the systems that were scanned"
+          report_id:
+            type: "integer"
+            format: "int64"
+            description: "The insights report identifier"
+            example: 1
+          report_type:
+            type: "string"
+            description: "The report type"
+            example: "insights"
+          report_version:
+            type: "string"
+            description: "The report version"
+            example: "1.0.0.afdsgev"
+          report_platform_id:
+            type: "string"
+            description: "Globally unique uuid for report"
+            example: "be3075ac-84d3-4b62-9f5c-a418a36f802d"
+          report_slices:
+            type: object
+            properties:
+              system_platform_id:
```

Comment:
instead of `system_platform_id` wouldn't this be `report_slice_id`?

Reply:
Take a look again.